packages/slice-machine/lib/builders/SliceBuilder/SetupDrawer/Stepper.tsx

```diff
@@ -0,0 +1,87 @@
+import * as steps from "./steps";
+
+import React from "react";
+import useSliceMachineActions from "@src/modules/useSliceMachineActions";
+import { useSelector } from "react-redux";
+import { SliceMachineStoreType } from "@src/redux/type";
+import {
+  selectOpenedStep,
+  selectSetupStatus,
+  selectUserHasAtLeastOneStepMissing,
+  selectUserHasConfiguredAllSteps,
+} from "@src/modules/preview";
+
+import { isLoading } from "@src/modules/loading";
+import { LoadingKeysEnum } from "@src/modules/loading/types";
+import { Frameworks } from "@slicemachine/core/build/src/models";
+import { getFramework, selectPreviewUrl } from "@src/modules/environment";
+
+interface Props {
+  framework: Frameworks | undefined;
+}
+
+export default function Stepper({ framework }: Props): React.ReactElement {
+  if (framework !== Frameworks.next && framework !== Frameworks.nuxt) {
```

Comment:
Actually we can use previewIsSupported here

Reply:
done